plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -144,7 +144,8 @@ private MLCommonsSettings() {}
                     "^https://runtime\\.sagemaker\\..*[a-z0-9-]\\.amazonaws\\.com/.*$",
                     "^https://api\\.openai\\.com/.*$",
                     "^https://api\\.cohere\\.ai/.*$",
-                    "^https://bedrock-runtime\\..*[a-z0-9-]\\.amazonaws\\.com/.*$"
+                    "^https://bedrock-runtime\\..*[a-z0-9-]\\.amazonaws\\.com/.*$",
+                    "^https://bedrock-agent-runtime\\..*[a-z0-9-]\\.amazonaws\\.com/.*$"
```

Comment:
should we combine these two bedrock endpoint into one?

Reply:
Do you mean regex like the following?
`^https://(?:bedrock-runtime|bedrock-agent-runtime)\\..*[a-z0-9-]\\.amazonaws\\.com/.*$
`

But I think the current code is more readable, so I prefer remain unchanged.

Reply:
I mean bedrock.* to include all bedrock related, so we don't need to add more if there has another bedrock-xyz. I think it's ok to keep as two for now and merge into one if we have more bedrock endpoints comes after.